CIScripts/Test/Utils/IisTcpTestDockerImageBuild.ps1

```diff
@@ -0,0 +1,19 @@
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+$DockerfilePath = "$PSScriptRoot\..\..\DockerFiles\iis-tcptest\Dockerfile"
+
+function Initialize-IisTcpTestDockerImage  {
```

Comment:
[Refinement] We think that this function could be more generic and used for any docker image. Image name could be passed as parameter. In this case it would be 'iis-tcptest'. This function can assume that all docker files are present in $PSScriptRoot\..\..\DockerFiles\.